torch/distributed/rendezvous.py

```diff
@@ -194,7 +197,9 @@ def _error(msg):
 
     rank = int(query_dict["rank"])
     world_size = int(query_dict["world_size"])
-    use_libuv = query_dict.get("use_libuv", "0") == "1"
+    # libuv is the default backend for TCPStore. To enable the non-libuv backend,
+    # user can explicitly specify ``use_libuv=0`` in the URL parameter.
+    use_libuv = query_dict.get("use_libuv", os.environ.get("USE_LIBUV", "1")) == "1"
```

Comment:
Do we need to update any of the public docs on a) type of backend used by default b) option of overriding the behavior via `USE_LIBUV`?

Reply:
I don't think so (at least not urgent before the release cutoff) because this change does not require user awareness (e.g. users don't need to change code to adapt the change). The only thing that needs addressed is the `TCPStore` init function signature. We need to add `use_libuv` argument in the description but let's have it in another PR.